k8s/base/deployment-config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps.openshift.io/v1
```

Comment:
Note that `DeploymentConfig` is an openshift-specific data type, and
the OpenShift docs generally recommend using `Deployment` objects
instead, which are part of core Kubernetes:

> Both Kubernetes Deployment objects and OpenShift Container Platform-provided DeploymentConfig objects are supported in OpenShift Container Platform; however, it is recommended to use Deployment objects unless you need a specific feature or behavior provided by DeploymentConfig objects.

(from "[Understanding Deployment and DeploymentConfig objects][1]")

[1]: https://docs.openshift.com/container-platform/4.7/applications/deployments/what-deployments-are.html#deployments-comparing-deploymentconfigs_what-deployments-are

Reply:
This is what was being used in the `build.py` script. I tried to stay pretty close to it. We can change this in a follow-up patch if needed.